test/test_time.py

```diff
@@ -0,0 +1,177 @@
+import sbol3
+import paml
+import unittest
+import tyto
+from paml_check.paml_check import check_doc
+
+class TestTime(unittest.TestCase):
+    def test_difference(self):
+        #############################################
+        # set up the document
+        print('Setting up document')
+        doc = sbol3.Document()
+        sbol3.set_namespace('https://bbn.com/scratch/')
+
+
+        #############################################
+        # Create the Difference
+        print('Creating Difference')
+        t1 = paml.TimeVariable(value=sbol3.Measure(0, tyto.OM.time))
```

Comment:
time is not a unit; time is a quantity, like length. You want a unit here like hours or seconds.